.ci/cico_build_deploy_test_rhche.sh

```diff
@@ -56,7 +56,7 @@ else
   echo "Switching to project failed, probably not exists [$?]. Creating..."
   oc new-project "${PROJECT_NAMESPACE}" --display-name="PR ${RH_PULL_REQUEST_ID} - Automated Deployment" > /dev/null 2>&1
 fi
-oc policy add-role-to-user edit ibuziuk -n $PROJECT_NAMESPACE
+oc policy add-role-to-user edit Katka92 ScrewTSW rhopp tomgeorge ibuziuk amisevsk davidfestal skabashnyuk -n $PROJECT_NAMESPACE
```

Comment:
are you sure that all the users are provisioned against the new cluster?
Could you add only Hosted Che team members for now.

Reply:
I can remove Angel and David, but I would leave Sergii there

Reply:
could you add only people that **need** access for now
I would, in general, prefer that anyone who needs access will submit a separate PR. why do you want to add everyone from the previous list if you even do not know if a) you do not know if people are provisioned on the cluster b) people need this access